build/Settings.cs

```diff
@@ -56,5 +57,73 @@ private static string config(string @default = null, [CallerMemberName] string k
         public static string CommitId => config(null, "Build.SourceVersion") ?? config("N /A", "APPVEYOR_REPO_COMMIT");
 
         public static string BuildArtifactsStorage => config(null);
+
+        public class SignInfo
+        {
+            public static string AzureSigningConnectionString => config(null, "AzureBlobSigningConnectionString");
+            public static readonly string AzureToSignContainerName = "azure-functions-cli";
+            public static readonly string AzureSignedContainerName = "azure-functions-cli-signed";
+            public static readonly string AzureSigningJobName = "signing-jobs";
+            public static readonly string ToSignZipName = $"{BuildNumber}-authenticode.zip";
+            public static readonly string ToSignThirdPartyName = $"{BuildNumber}-third.zip";
+            public static readonly string ToSignDir = "unsigned";
+            public static readonly string SignedDir = "signed";
+            public static readonly string Authenticode = "SignAuthenticode";
+            public static readonly string ThirdParty = "Sign3rdParty";
+            public static readonly string[] RuntimesToSign = new[] {"min.win-x86"};
+            public static readonly string[] filterExtenstionsSign = new[] { ".json", ".spec", ".cfg", ".pdb", ".config", ".nupkg", ".py" };
+
+            public static readonly string[] authentiCodeBinaries = new[] {
+                "DurableTask.AzureStorage.Internal.dll",
+                "DurableTask.Core.Internal.dll",
+                "func.dll",
+                "func.exe",
+                "func.pdb",
+                "Microsoft.Azure.AppService.Proxy.Client.dll",
+                "Microsoft.Azure.AppService.Proxy.Common.dll",
+                "Microsoft.Azure.AppService.Proxy.Runtime.dll",
+                "Microsoft.Azure.WebJobs.dll",
+                "Microsoft.Azure.WebJobs.Extensions.Http.dll",
+                "Microsoft.Azure.WebJobs.Host.dll",
+                "Microsoft.Azure.WebJobs.Host.Storage.dll",
+                "Microsoft.Azure.WebJobs.Logging.ApplicationInsights.dll",
+                "Microsoft.Azure.WebJobs.Logging.dll",
+                "Microsoft.Azure.WebJobs.Script.dll",
+                "Microsoft.Azure.WebJobs.Script.Grpc.dll",
+                "Microsoft.Azure.WebJobs.Script.WebHost.dll",
+                "Microsoft.Azure.WebSites.DataProtection.dll",
+                Path.Combine("templates", "itemTemplates.2.0.10328.nupkg"),
```

Comment:
templates would keep changing every release. This would break on every release.

Reply:
Also templates are already signed.

Reply:
Ok, right now, any files from the list that are not present in the Filesystem is ignored. I can remove this in the PR. Unless this is blocking, we should go ahead with the update.

Additionally, I am not signing nupkgs yes, I filter out certain extensions defined in this Settings file.